frontend/src/Directions.tsx

```diff
@@ -0,0 +1,51 @@
+import { latLng, LatLng } from "leaflet";
+import { RideRoute } from "./pages/CreateRide";
+
+/**
+ * MapQuest Open Directions API functions and components.
+ */
+const MQ_DIR_URI = "https://open.mapquestapi.com/directions/v2/route";
+const MQ_KEY = "zrK0kZ2o9WcxfTJpYYWaZ9uYHYSZvcyC";
```

Comment:
Is this meant to be a secret?

Reply:
Yes, I don't know how to properly do that and I haven't given them my credit card yet.